zaza/openstack/charm_tests/test_utils.py

```diff
@@ -728,16 +728,10 @@ def setUpClass(cls):
     """
 
     @classmethod
-    def setUpClass(cls, restart_config_file, test_service, restart_package,
-                   restart_package_service, application_name):
+    def setUpClass(cls, restart_package, restart_package_service,
+                   application_name, deferred_hook='config-changed'):
```

Comment:
`deferred_hook` is in fact a string that will be searched in the workload status msg and I think this deserves a docstring

Reply:
Oh it's gone meanwhile, sorry. I have been reviewing commit by commit. Let me take a final look